tests/tests/busola-create-a-deployment.spec.js

```diff
@@ -0,0 +1,162 @@
+/// <reference types="cypress" />
+import config from '../config';
+import 'cypress-file-upload';
+
+const ADDRESS = config.localDev
+  ? `http://localhost:4200`
+  : `https://busola.${config.domain}`;
+
+const random = Math.floor(Math.random() * 1000);
+const NAMESPACE_NAME = `b-busola-test-${random}`;
+
+const DOCKER_IMAGE = 'eu.gcr.io/kyma-project/pr/orders-service:PR-162';
+const DEPLOYMENT_NAME = 'orders-service';
+
+context('Busola - Create a Deployment', () => {
+  const getLeftNav = () => cy.get('nav[data-testid=semiCollapsibleLeftNav]');
+
+  before(() => {
+    cy.visit(ADDRESS)
```

Comment:
You need to adjust login flow. Example is here: https://github.com/kyma-project/busola/blob/main/tests/tests/busola-smoke-tests.spec.js#L18